### DISCUSSION TITLE
Create Quadrants (Grid) in the Presentation Slide to Add Elements

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#1552] [Q&A]
The great thing about WYSIWYG editors is the ability to have a flexible location for each element.
This is both for creating interesting slides and a better use of space.

An idea to accomplish a similar effect was to have a class which creates a grid on the slide (Maybe multiple grids with different size) and allows easily set an object inside it.
It should also allow chose the scaling of the elements within the grid (Similar to stretching, with ratio preservation and without).

Is there a way to achieve this? Could it be added a s a feature?

**Comment by RoyiAvital** [comment]
I saw that `.absolute` was added to documentation which is almost there.

It solves the problem for images, but can we create other containers like that as well?
Something like the shapes in PowerPoint with added test inside them?

**Comment by mcanouil** [comment]
It's HTML, you can use the grid system classes of Boostrap.

**Comment by RoyiAvital** [comment]
@mcanouil , Indeed. But this logic leads to "I can do everything in HTML".
The whole idea of Quarto is abstracting it, right?

**Reply by mcanouil** [reply]
It cannot abstract everything obviously.
To note for regular HTML format, Quarto's documentation shows Grid css classes from Boostrap used with fenced divs, see <https://quarto.org/docs/output-formats/page-layout.html#css-grid>.
